requirements-test.txt

```diff
@@ -1,5 +1,3 @@
 coveralls
-isort
-flake8
-openwisp-utils[qa]>=0.2.1
+openwisp-utils[qa]>=0.3.2
```

Comment:
set this to `0.4.0`

Reply:
Done, but still it's showing the same conflict :expressionless: